pyaedt/edb.py

```diff
@@ -3022,3 +3230,32 @@ def create_siwave_dc_setup(self, name=None):
         setup = SiwaveDCSimulationSetup(self, name)
         self._setups[name] = setup
         return setup
+
+    @pyaedt_function_handler()
+    def calculate_initial_extent(self, expansion_factor):
+        """Compute a float representing the larger number between the dielectric thickness or trace width
+        multiplied by the nW factor. The trace width search is limited to nets with ports attached.
+
+        Parameters
+        ----------
+        expansion_factor : float
+            The number of 'widths' multiplier
```

Comment:
```suggestion
            Value for the width multiplier (nW factor).
```
Not sure if I got this correct, but the original description seemed to lack detail.